lua/base16-colorscheme.lua

```diff
@@ -441,7 +441,7 @@ function M.setup(colors, config)
         hi['@string.special.symbol']    = '@symbol'
         hi['@string.regexp']            = '@string.regex'
         hi['@markup.strong']            = '@text.strong'
-        hi['@markup.italic']            = '@text.italic'
+        hi['@markup.italic']            = '@text.emphasis'
```

Comment:
Why not this?

```suggestion
        hi['@markup.italic']            = 'Italic'
```

Reply:
i saw that the others were using `@text.*`, so i picked `@text.emphasis`